cmake/examples.cmake

```diff
@@ -166,49 +170,61 @@ add_example(
 
 # Only add examples if set, otherwise we still need exmaples common for tools
 if( BGFX_BUILD_EXAMPLES )
-    # Add examples
-    set(
-        BGFX_EXAMPLES
-        00-helloworld
-        01-cubes
-        02-metaballs
-        03-raymarch
-        04-mesh
-        05-instancing
-        06-bump
-        07-callback
-        08-update
-        09-hdr
-        10-font
-        11-fontsdf
-        12-lod
-        13-stencil
-        14-shadowvolumes
-        15-shadowmaps-simple
-        16-shadowmaps
-        17-drawstress
-        18-ibl
-        19-oit
-        20-nanovg
-        21-deferred
-        22-windows
-        23-vectordisplay
-        24-nbody
-        25-c99
-        26-occlusion
-        27-terrain
-        28-wireframe
-        29-debugdraw
-        30-picking
-        31-rsm
-        32-particles
-        33-pom
-        34-mvs
-        35-dynamic
-        36-sky
-    )
-
-    foreach( EXAMPLE ${BGFX_EXAMPLES} )
-        add_example( ${EXAMPLE} )
-    endforeach()
+	# Add examples
+	set(
+	BGFX_EXAMPLES
+	00-helloworld
+	01-cubes
+	02-metaballs
+	03-raymarch
+	04-mesh
+	05-instancing
+	06-bump
+	07-callback
+	08-update
+	09-hdr
+	10-font
+	11-fontsdf
+	12-lod
+	13-stencil
+	14-shadowvolumes
+	15-shadowmaps-simple
+	16-shadowmaps
+	17-drawstress
+	18-ibl
+	19-oit
+	20-nanovg
+	21-deferred
+	22-windows
+	23-vectordisplay
+	24-nbody
+	25-c99
+	26-occlusion
+	27-terrain
+	28-wireframe
+	29-debugdraw
+	30-picking
+	31-rsm
+	32-particles
+	33-pom
+	34-mvs
+	35-dynamic
+	36-sky
+	#37-gpudrivenrendering

```

Comment:
It doesn't look like the shaders of 37-gpudrivenrendering doesn't compile anymore. Most likely an issue on bgfx side.

cd /home/mchiasson/git/build-bgfx.cmake-Debug && /home/mchiasson/git/build-bgfx.cmake-Debug/shaderc -f /home/mchiasson/git/bgfx.cmake/bgfx/examples/37-gpudrivenrendering/fs_gdr_instanced_indirect_rendering.sc -i /home/mchiasson/git/bgfx.cmake/bgfx/src -o /home/mchiasson/git/bgfx.cmake/bgfx/examples/runtime/shaders/gles/fs_gdr_instanced_indirect_rendering.bin --platform android --type fragment && /home/mchiasson/git/build-bgfx.cmake-Debug/shaderc -f /home/mchiasson/git/bgfx.cmake/bgfx/examples/37-gpudrivenrendering/fs_gdr_instanced_indirect_rendering.sc -i /home/mchiasson/git/bgfx.cmake/bgfx/src -o /home/mchiasson/git/bgfx.cmake/bgfx/examples/runtime/shaders/glsl/fs_gdr_instanced_indirect_rendering.bin --platform linux --type fragment --profile 120
Code:
---
    354: void main()
    355: {
    356: vec4 color = u_color[uint(v_materialID)];
    357: if (color.w < 1.0f)
    358: {
    359: if ( (int(gl_FragCoord.x) % 2) == (int(gl_FragCoord.y) % 2) )
    360: {
    361: discard;
    362: }
    363: }

>>> 364: gl_FragColor = vec4(color.xyz, 1.0);
>>>   1: ^

    365: }
---
Error: (364,1): error: `gl_FragColor' undeclared
(364,1): error: value of type vec4 cannot be assigned to variable of type error

Failed to build shader.
For now, I disabled 37-gpudrivenrendering since it can't generate properly for essl (however, it generates fine in glsl and spirv). I'll come back later to uncomment this once shaderc is fixed on bgfx, and that your submodules are updated.